subscribe_gnmi.go

```diff
@@ -461,6 +461,7 @@ func subscribegNMI(conn *grpc.ClientConn, jctx *JCtx, cfg Config, paths []PathsC
 		sub  = gnmi.SubscribeRequest_Subscribe{Subscribe: &subs}
 		req  = gnmi.SubscribeRequest{Request: &sub}
 		err  error
+		err1 error
```

Comment:
Should we move this inside the go routine ?

Reply:
done